src/vl-map-layer-style.js

```diff
@@ -28,11 +40,11 @@ export class VlMapLayerStyle extends vlElement(HTMLElement) {
    * @Return {string}
    */
   get color() {
-    return this.getAttribute('color') || 'rgba(2, 85, 204, 1)';
+    return this.getAttribute('color') || 'rgba(2, 85, 204, 0.8)';
```

Comment:
die default value is ook maar gewoon iets random?

Reply:
ik bedoel dan eerder de kleur dan dat ge het nu een beetje transparant hebt gemaakt.